### DISCUSSION TITLE
What package do I use to have vercel components at https://vercel.com/design?

### DISCUSSION THREAD

**Discussion by vasconx** [vercel/vercel#5801] [Help]
In the vercel design link, there is no reference to the components, just  "Button, Modal" etc ... Does anyone know if there is a specific module?
Because in my head it doesn’t make much sense to do all this if there’s no package behind it, it’s like I’m creating a recipe with an ingredient that doesn’t exist, a chapolate cake

**Comment by jacobhq** [comment]
Try [Geist UI](https://react.geist-ui.dev/). It used to be called Zeit UI from when Vercel was called Zeit.
